profiler/test/Datadog.Profiler.Native.Tests/ConfigurationTest.cpp

```diff
@@ -133,12 +133,7 @@ TEST(ConfigurationTest, CheckDefaultPprofDirectoryWhenVariableIsNotSet)
 {
     unsetenv(EnvironmentVariables::ProfilesOutputDir);
     auto configuration = Configuration{};
-    auto expectedValue =
-#ifdef _WINDOWS
-        WStr("C:\\ProgramData\\Datadog-APM\\Pprof-files\\DotNet");
```

Comment:
GetDefaultPprofDirectoryPath() in configuration.cpp file is using GetApmBaseDirectory() instead of these hardcoded values. On one hand it allows us to detect if we change that folder but is this what we want?